region/brinstar/blue/Morph Ball Room.json

```diff
@@ -1118,6 +1174,46 @@
         }}
       ]
     },
+    {
+      "link": [6, 1],
+      "name": "Pause Abuse Sidehopper Hit",
+      "requires": [
+        "h_pauseAbuseMinimalReserveRefill"
+      ]
+    },
+    {
+      "link": [6, 1],
+      "name": "Pause Abuse Quick Sidehopper Kill",
+      "requires": [
+        "h_pauseAbuseMinimalReserveRefill",
+        {"or": [
+          "Plasma",
+          {"enemyKill": {
+            "enemies": [["Sidehopper", "Sidehopper", "Sidehopper"]],
+            "explicitWeapons": ["Missile", "Super"]
+          }}
+        ]}
+      ],
+      "clearsObstacles": ["C"]
+    },
+    {
+      "link": [6, 1],
+      "name": "Pause Abuse Pseudo Screw Sidehopper Kill",
+      "requires": [
+        "canPseudoScrew",
+        "h_pauseAbuseMinimalReserveRefill",
```

Comment:
I pseudo screw the first one then kill it and the top guy with pea shooter while taking a second hit. Crouching in the corner sometimes gets a third hit, but with morph it wouldnt. The last guy can be killed with a pseudo and pause abuse or morph dodge or the jump dodge. 

I'll add another hit, and rework the second part, because fighting the first two can easily get out of hand.